src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -1,8 +1,9 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
This is quite a complicated diff, is there anything other than formatting changes? Does it work?

Reply:
This was an attempt to get around an error but it didn't work. I just left it in the draft to show what I'd tried